libraries/adsk/adsklib/colorcorrect.mtlx

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<materialx version="1.38" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <nodedef name="ND_colorcorrect" node="colorcorrect1" nodegroup="adjustment">
+    <output name="outColor" type="color3" />
+    <output name="outAlpha" type="float" />
+    <input name="AlphaClampAndGamma_inlow" type="float" value="0" />
```

Comment:
These are not proper names but ones generated by call into core code to generate this.
Just an example for now.